src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -1155,7 +1155,7 @@ public sealed partial class JsonPropertyInfoValues<T>
         public System.Action<object, T?>? Setter { get { throw null; } init { } }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-    public partial class JsonTypeInfo
+    public abstract partial class JsonTypeInfo
```

Comment:
For changes updating public API I would _strongly recommend_ consolidating everything in a single PR exposing the changes as approved in https://github.com/dotnet/runtime/issues/63686#issuecomment-1125339465. This will give us the opportunity to review the changes in unison and will let us make sure nothing is left out.

Tactically speaking, what necessitates breaking up the implementation of this feature in multiple PRs?